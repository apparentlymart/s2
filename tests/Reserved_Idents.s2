#-*-c++-*-

layerinfo "type" = "core";
layerinfo "majorversion" = 0;

property string blah;
set "blah" = "foo";

function main ()
{
  var string n;
  $n = "foo";

  var int size;

  var string[] a;
  $a[0] = "a";
  $a[1] = "b";
  $a[2] = "c";
  $a[3] = "d";

  $size = size $a;

  print "n=$n, size=$size\n";
}



