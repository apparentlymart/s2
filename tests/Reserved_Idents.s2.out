n=foo, size=4
